utils/debugdump.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+// These are functions specifically used for Debugging purposes.
+// These should never go into Released versions
```

Comment:
I do not like having the comment 
```
These should never go into Released versions
``` 
when this WILL be going into the release version!

Reply:
just have comment
```
// functions specifically used for Debugging purposes.
```